libs/libopen62541/Makefile

```
@@ -0,0 +1,218 @@
+#
+# Copyright (C) 2023 Michele Primavera
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libopen62541
+PKG_VERSION:=1.3.5
+PKG_RELEASE:=1
+#PKG_USE_MIPS16:=0
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/open62541/open62541/archive/
+PKG_HASH:=308aa8845f9cc3a99427636d2282e9bed2a26fbc8a404fe2ad1699146e4f4342
+PKG_BUILD_DIR:=$(BUILD_DIR)/open62541-$(PKG_VERSION)
+
+PKG_LICENSE:=MPLv2
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+CMAKE_INSTALL:=1
+
+define Package/libopen62541/config
+        source "$(SOURCE)/Config.in"
+endef
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+ifeq ($(CONFIG_HAS_MIPS16),y)
+  TARGET_CFLAGS += -fstack-check=generic
+endif
+
+define Package/libopen62541
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=An OPC UA library
+  URL:=https://open62541.org/
+  DEPENDS:=+librt +zlib +libstdcpp +libpthread +libatomic +LIBOPEN62541_MBDEDTLS:libmbedtls +LIBOPEN62541_OPENSSL:libopenssl @!LINUX_2_6
```

```suggestion
  DEPENDS:=+librt +zlib +libstdcpp +libpthread +libatomic +LIBOPEN62541_MBDEDTLS:libmbedtls +LIBOPEN62541_OPENSSL:libopenssl
```

linux 2.6 support was dropped long long ago